pinpoint-hbase/hbase-site-standalone.xml

```diff
@@ -0,0 +1,42 @@
+<configuration>
+    <property>
+        <name>hbase.rootdir</name>
+        <value>file:///home/pinpoint/hbase</value>
+    </property>
+    <property>
+        <name>hbase.cluster.distributed</name>
+        <value>true</value>
+    </property>
+     <property>
+        <name>zookeeper.znode.parent</name>
+        <value>/hbase</value>
+    </property>
+    <property>
+        <name>hbase.zookeeper.quorum</name>
+        <value>zoo1,zoo2,zoo3</value>
+    </property>
+    <property>
+	<name>hbase.zookeeper.property.clientPort</name>
+	<value>2181</value>
+    </property> 
+    <property>
+        <name>hbase.master.port</name>
+        <value>60000</value>
+    </property>
+    <property>
+        <name>hbase.regionserver.port</name>
+        <value>16201</value>
+    </property>
+    <property>
+        <name>hbase.regionserver.hostname</name>
+        <value>localhost</value>
+    </property>
+    <property>
+        <name>hbase.regionserver.ipc.address</name>
+        <value>0.0.0.0</value>
+    </property>
+    <property>
+	<name>hbase.master.ipc.address</name>
+	<value>0.0.0.0</value>
+    </property>
```

Comment:
Is this necessary when the hbase is in stand-alone mode?

Reply:
I think it is. 
In order to pinpoint collector and web module that run in host need to know ipc address to connect hbase in docker container.

I had a wrong choice on the word 'stand-alone'. 
It is more related to docker container not the hbase itself.

Reply:
If you are using not referring to stand-alone hbase. What is this docker file for?

Reply:
This is for the one runs pinpoint collector and web module in IDE or host environment, and want to run hbase as container. because running with hbase binary is too troublesome.

Reply:
@cwJohnPark

I've understood your intentions. But with the below configuration, this will ONLY work IF HBase is on localhost. 
```
<property>
        <name>hbase.regionserver.hostname</name>
        <value>localhost</value>
</property>
```

Solution for the below
> This is for the one runs pinpoint collector and web module in IDE or host environment
should be taken care of by adding the hbase containers IP address in the host file.

I know adding a line in the host file may be annoying. But since the reason for this is due to the networking architecture of docker containers. I couldn't find any appropriate solution for this yet.
Please let me know what you think about this, and welcome if there is any better solution.